File: chanom.cpp
#include <iostream>
#include <stdlib.h>
#include <stdio.h>
#include <algorithm>
#include <stdio.h>
#include <time.h>

using namespace std;

static int getRandomInt(int m) {
	srand(time(NULL));
	int randomOut = rand() % m + 1;
	return randomOut;
}

int main() {
	int handsome;
	handsome = getRandomInt(100);
	cout << "Chanom is only handsome : " << handsome << "\% of Me!";
	return 0;
}

File: chanom.java
import java.lang.Math;

class chanom {
	static double getRandomInt(int m) {
		double randomOut = Math.floor(Math.random() * m) + 1;
		return randomOut;
	};
	public static void main(String[] args) {
		double handsome;
		handsome = getRandomInt(1);
		System.out.println("Chanom is only handsome : "+ handsome + "% of Me!");
	}
}